tarofreighter/parcel.go

```diff
@@ -225,7 +225,19 @@ type sendPackage struct {
 	// anchored by each output on the transfer transaction.
 	NewOutputCommitments taroscript.SpendCommitments
 
-	// SendPkt is the PSBT that will complete the transfer.
+	// FundedPkt is the PSBT that was funded by the lnd internal wallet.
+	// This will not be updated once the PSBT is signed and finalized in
+	// order to keep the change output information around that is needed for
+	// creating the exclusion proofs.
+	FundedPkt *psbt.Packet
+
+	// FundedPktChangeIdx is the change index for any BTC change output that
+	// was created when anchoring the send. This is -1 if no change output
+	// was created.
+	FundedPktChangeIdx int32
```

Comment:
Isn't this already returned along side the lnd response for `FundPSBT`?

Reply:
Ah, you're right. I changed things a bit to re-use that.